utils/appliance/__init__.py

```diff
@@ -1542,15 +1561,15 @@ def host_id(self, hostname):
 
     def get_yaml_config(self, config_name):
         if config_name == 'vmdb':
-            writeout = store.current_appliance.ssh_client.run_rails_command(
```

Comment:
This and the instance below might be causing some crazy issues in multi-appliance setups. Potentially in parallelizer as well, not sure if we operate on multiple appliances without switching context anywhere in there...